README.md

```diff
@@ -32,13 +27,24 @@ they wish to update.
 ## Start
 
 ```javascript
-var Logger = require('r7insight_node');
+var { Logger } = require('r7insight_node');
 
-var logger = new Logger({ token: 'myAccessToken' , region: 'myRegion'});
+var logger = new Logger({ token: '<token>' , region: '<region>'});
 
-logger.warning('The kittens have become alarmingly adorable.')
+logger.warning("I'll put this over here, with the rest of the fire.");
 ```
 
+## Development
```

Comment:
I still don't get why we have to document this. Do we realy want people that don't know how Node works to make changes to this? 😅

Reply:
I don't know how node works 😄 i think it's nice as a small, easy get started guode